test/lib/dependency-versions-test.ts

```diff
@@ -323,73 +330,51 @@ describe('Utils | dependency-versions', function () {
       const packageJson2: PackageJson = JSON.parse(packageJson2Contents);
 
       // foo
-      strictEqual(
-        packageJson1.dependencies && packageJson1.dependencies.foo,
-        '^2.0.0',
-        'updates the package1 `foo` version to the highest version'
```

Comment:
I really dislike how jest assertions cannot have custom error messages. It makes the tests much less understandable in situations like this. So I would actually prefer we keep using the node assertions in these places where custom error messages are provided (usually when many assertions are present inside the same `it`).

Reply:
Would it be better to keep jest assertions, and convert the error messages to comments?

Reply:
Sure, I would settle for converting the assertion message to a a comment for these equality assertions.